### PR TITLE
Fix: Improve docs and mobile style of api docs

### PR DESCRIPTION
**Pull request by agustif**
## What does this PR do?
 adds servers in openapi, remove hack in snippets, update deps make dynamic import to use latests swagger ui deps
improve mobile styles

### CODE REVIEW

File: apps/swagger/styles/globals.css
Comment (zomars) [inline apps/swagger/styles/globals.css:18]: Let's remove comments

**Review by zomars — APPROVED** [review]
Code looks good. Let's see why vuelva checks are failing

**Review by PeerRich — APPROVED** [review]

File: apps/swagger/styles/globals.css
Comment (agustif) [reply apps/swagger/styles/globals.css]: Done! Also removed the ugly scrollbar in cusotm css for now, will update/fix in another PR

File: package.json
Comment (zomars) [inline package.json:20]: We don't have a `dev` script on prisma. @agustif

**Review by zomars — CHANGES_REQUESTED** [review]
Let's check the type errors

File: package.json
Comment (agustif) [reply package.json:20]: Right @zomars, moved it from web to prisma and didn't thought well.

What do you think it's best, to run the whole web before api? We only require the docker/database to be available not really web...
Thoughts?

Comment (zomars) [reply package.json:20]: Just remove web if we don't need it

Comment (agustif) [reply package.json]: done!

**Comment by agustif** [comment]
> Code looks good. Let's see why vuelva checks are failing

Was missing to add iframe-react-resizer as dep to docs, added now, all checks should pass!
Can we merge after? adding automerge waiting for your approval @zomars

**Comment by agustif** [comment]
Removing json-schema from packages/prisma as won't be used/auto genereated anymore less is more

**Review by zomars — APPROVED** [review]

**Pull request merged**
